webapp/portlet/src/main/webapp/vue-apps/common/initComponents.js

```diff
@@ -54,6 +55,7 @@ const components = {
   'unread-badge': UnreadBadge,
   'alert-notifications': Notifications,
   'ripple-hover-button': RippleHoverButton,
+  'translation-drawer': TranslationDrawer,
```

Comment:
It's already re-usable, but you just have to add a dependency to the module inside gatein-resources.xml like here:

https://github.com/Meeds-io/social/blob/651de4582bfd9acd6ea4526455b9b9188cf80c03/webapp/portlet/src/main/webapp/WEB-INF/gatein-resources.xml#L1400-L1402

Reply:
yes @boubaker I see but in our case we need only to use translation-drawer component from our side not the whole translation-field component, can we make translation-drawer reusable component from commons ?

Reply:
ok we will adapt our work by using translation-field, thanks @boubaker